.github/workflows/develop.yml

```
@@ -0,0 +1,126 @@
+name: orvit-server-dev
+
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    outputs:
+      short_sha: ${{ steps.meta.outputs.short_sha }}
```

actionlint (1.7.10)

[error] property "meta" is not defined in object type {}

(expression)